index.bs

```diff
@@ -2307,6 +2307,10 @@ same as the [=window handle=].
 Each [=/browsing context=] also has an <dfn>associated storage partition</dfn>,
 which is the [=storage partition=] it uses to persist data.
 
+Each [=/browsing context=] also has an associated <dfn>original
+opener</dfn>, which is a [=/browsing context=] that caused the creation
```

Comment:
NIT: we don't have a rule for this, but it would be really helpful to avoid line breaks in the tags. Otherwise editing spec become more chellenging.